cms/utils/setup.py

```diff
@@ -38,6 +42,10 @@ def validate_settings():
                 'django.template.context_processors.request' not in context_processors):
             raise ImproperlyConfigured("django CMS requires django.template.context_processors.request in "
                                        "'django.template.backends.django.DjangoTemplates' context processors.")
+    # Warn on empty CMS_INTERNAL_IPS
+    if not get_cms_setting('INTERNAL_IPS'):
+        warn('The setting `CMS_INTERNAL_IPS` was not defined or is empty, this '
```

Comment:
is this really necessary? I think it can get very annoying as usually this level of configuration is not needed. Also systems where ips are dynamically assigned can't really use this feature so I'd say we don't need to warn about it.
Documentation should be enough.